WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -614,6 +614,14 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         controller.modalPresentationStyle = .overCurrentContext
         self.present(controller, animated: true)
     }
+
+    func gutenbergDidRequestGetStarterPageTemplatesTooltipShown() -> Bool {
+       return GutenbergSettings().getStarterPageTemplatesTooltipShown()
+    }
+
+    func gutenbergDidRequestSetStarterPageTemplatesTooltipShown(_ tooltipShown: Bool) {
+       GutenbergSettings().setStarterPageTemplatesTooltipShown(tooltipShown)
+    }
```

Comment:
This style of calling `GutenbergSettings` seems to be more in line with refactoring these methods to be static methods or maybe share the instance of `GutenbergSettings` somewhere. Right now it looks like it's only used in one spot as a lazy-loaded var
https://github.com/wordpress-mobile/WordPress-iOS/blob/617a7a3092c8184bc14bf9ef572297a58ab021f0/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift#L226-L228

This might be a good time to refactor that to share the instance in this file such as:
```
lazy var gutenbergSettings: GutenbergSettings = { 
     return GutenbergSettings()
 }() 
```

Then these methods become:
```suggestion

    func gutenbergDidRequestGetStarterPageTemplatesTooltipShown() -> Bool {
       // return gutenbergSettings.starterPageTemplatesTooltipShown // depending on if you use the other suggestion or not
       return gutenbergSettings.getStarterPageTemplatesTooltipShown()
    }

    func gutenbergDidRequestSetStarterPageTemplatesTooltipShown(_ tooltipShown: Bool) {
        // gutenbergSettings.starterPageTemplatesTooltipShown = tooltipShown // depending on if you use the other suggestion or not
       gutenbergSettings.setStarterPageTemplatesTooltipShown(tooltipShown)
    }
```

Reply:
Sounds good! I'll refactor that part and update the code.